server/app/views/analyses/plot_scatter.html.erb

```diff
@@ -107,12 +107,15 @@
 
           var color = d3.scale.category10();
 
+          var excludedData = ['wwr_west_static', 'data_point_uuid', '_id', 'name', 'run_start_time', 'run_end_time', 'status', 'status_message'];
```

Comment:
We need to populate this list from some sort of metadata item in the variable list.

* remove wwr_west_static -- that is way to custom. I realize that is mine from years ago :(